Feature/co 732 time formatting csv

jaredkosanovic opened the pull request.

Commit:
created helper function for formatting dates and times for csv

Commit:
added some unit tests

Commit:
consistent quotes

Commit:
timezone as a config value

the timezone for the csv endpoint (the only endpoint that won't return times in UTC) depends on the environment, thus set it in configuration.yaml

floppyears commented on src/main/groovy/edu/oregonstate/mist/osuevents/resources/CSVHelperFunctions.groovy:
```diff
@@ -0,0 +1,23 @@
+import java.time.format.DateTimeFormatter
+
+class CSVHelperFunctions {
+    public static String getCSVDate(ZonedDateTime dateTime, ZoneId backendTimeZone) {
```
This method could benefit from a comment.

floppyears commented on src/main/groovy/edu/oregonstate/mist/osuevents/resources/CSVHelperFunctions.groovy:
```diff
@@ -0,0 +1,23 @@
+        dateTime.format(csvDateFormatter)
+    }
+
+    public static String getCSVTime(ZonedDateTime dateTime, ZoneId backendTimeZone) {
```
These two methods are not DRY. How about having a private helper method. It would take the two parameters that these two methods already accept, but also take in a pattern string. Both of these methods would then just call the private method and return the output right away.

floppyears replied:
These methods also need a simple unit test to go along with them.

Commit:
add more tests, classrule

removed default values in errorMessages and instead allowed unit tests to get data from resource.properties by declaring a @ClassRule. Also added some more tests.

Commit:
make it DRY

jaredkosanovic committed:
Merge pull request #6 from osu-mist/feature/CO-689-create-unit-tests

Feature/co 689 create unit tests

Commit:
added unit test for csv time methods

Commit:
merge from develop, add default timezone

added default timezone as UTC for EventsResource
